docs/settings.md

```diff
@@ -82,6 +82,7 @@ Legend:
 | `bridge_bot_password`                | string                                                           |            | No   |        | Automatically generated password of the matterbridge bot user profile                                                                                                             |
 | `default_attachment_folder`          | string                                                           | `/Talk`    | No   |        | Specify default attachment folder location                                                                                                                                        |
 | `start_calls`                        | int                                                              | `0`        | Yes  | 🖌️    | Who can start a call, see [constants list](constants.md#start-call)                                                                                                               |
+| `max_call_duration`                  | int                                                              | `0`        | No   | ️      | Maximum duration of a call in seconds, 0 for unlimited                                                                                                                            |
```

Comment:
```suggestion
| `max_call_duration`                  | int                                                              | `0`        | No   | ️       | Maximum duration of a call in seconds, 0 for unlimited                                                                                                                            |
```

Reply:
the diff is lying due to the emoji taking up 2 spaces in bytes, it's correct like done with other rows

Reply:
Hmm, indeed, it has 8 characters. But it is definitely different. I'm not comparing with the row with an emoji.

Reply:
Replacing [`񥀹`](https://unicode-explorer.com/c/65039) with a space